front/app/containers/LandingPage/SignedInHeader.tsx

```diff
@@ -233,7 +233,11 @@ const CompleteProfileIcon = styled(Icon)`
   margin-left: -3px;
 `;
 
-export const Text = styled.div``;
+export const Text = styled.div`
```

Comment:
Perhaps this entire component can be replaced by our `Text` component which can be imported from the component library and supports rtl by default?

Reply:
I tried it, had some weird behaviour. `Text` has some other assumptions associated with it compared to a clean div.